demo-app/src/app/pagination/pagination.component.html

```diff
@@ -0,0 +1,194 @@
+<h3>Pagition</h3>
+<div class="section">
+  <app-code-snippet>import &#123; MzPaginationModule &#125; from 'ng2-materialize'</app-code-snippet>
+</div>
+<div class="section">
+  <h5 class="light">Basic pagination</h5>
+
+  <div class="row m-valign-wrapper">
+    <mz-pagination class="col s12 m6"
+      [itemsPerPage]="10"
+      [totalItems]="100"
+    >
+    </mz-pagination>
+
+    <p class="col s12 m6">
+      By setting <code class="language-markup">itemsPerPage</code> and <code class="language-markup">totalItems</code> input property on <code class="language-markup">mz-pagination</code> component, the number of available pages will automatically be calculated.
+    </p>
+</div>
+
+<div class="section">
+  <h5 class="light">Active page</h5>
+
+  <div class="row m-valign-wrapper">
+    <mz-pagination class="col s12 m6"
+      [currentPage]="5"
+      [itemsPerPage]="10"
+      [totalItems]="100"
+    >
+    </mz-pagination>
+
+    <p class="col s12 m6">
+      Use <code class="language-markup">currentPage</code> input property to select a specific page when rendering.
+    </p>
+</div>
+
+<div class="section">
+  <h5 class="light">Displayed pages</h5>
+
+  <div class="row m-valign-wrapper">
+    <mz-pagination class="col s12 m6"
+      [itemsPerPage]="10"
+      [maxPageButtons]="10"
+      [totalItems]="100"
+    >
+    </mz-pagination>
+
+    <p class="col s12 m6">
+      Optionally, you can change the number of displayed pages by using <code class="language-markup">maxPageButton</code> proprety. When no value is provided the number of displayed pages is defaulted to 5.
+    </p>
+</div>
+
+<div class="section">
+  <h5 class="light">Basic pagination</h5>
+
+  <div class="row m-valign-wrapper">
+    <mz-pagination class="col s12 m6"
+      [itemsPerPage]="10"
+      [totalItems]="100"
+    >
+    </mz-pagination>
+
+    <p class="col s12 m6">
+      By setting <code class="language-markup">itemsPerPage</code> and <code class="language-markup">totalItems</code> input property on <code class="language-markup">mz-pagination</code> component, the number of available pages will automatically be calculated.
+    </p>
+</div>
+
+<div class="section">
+  <h5 class="light">First and last page button</h5>
+
+  <div class="row m-valign-wrapper">
+    <mz-pagination class="col s12 m6"
+      [enableFirstAndLastPageButtons]="true"
+      [itemsPerPage]="10"
+      [totalItems]="100"
+    >
+    </mz-pagination>
+
+    <p class="col s12 m6">
+      You can show first page and last page buttons by setting <code class="language-markup">enableFirstAndLastButtons</code> input property to <code class="language-markup">true</code>.
```

Comment:
`enableFirstAndLastButtons` does not fit `@Input enableFirstAndLastPageButtons` property name in the component